new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItDomainInPV.java

```diff
@@ -573,20 +596,291 @@ public void testDomainInPvUsingWdt() {
         .isTrue();
   }
 
+
   /**
-   * Uninstall Nginx.
-   * The cleanup framework does not uninstall Nginx release.
-   * Do it here for now.
+   * Test domain status updated when introspector run triggered by introSpectVersion.
```

Comment:
Should we have a test to just update the introSpectVersion with out any config change.

Reply:
we can but what is the purpose of it?